configuration/agent-settings.md

```diff
@@ -10,43 +10,52 @@ description: >-
 
 | Key | Description | Default  | Required |
 | :--- | :--- | :--- | :--- |
-| ac\_buffer\_size | Anomaly Capture buffer size, represents the number of log lines to capture during an anomaly capture | 125 | No |
 | tag | User defined tag to describe the server type, environment, etc. for the host the agent will be deployed on. Metrics output by the agent to backend systems will have this tag included. Example: prod\_payments | Edge | No |
-| disable\_printer | Disable displaying agent process results on screen. Enabled by default | false | No |
+| anomaly\_capture\_size | Anomaly Capture buffer size represents the number of log lines to capture during an anomaly capture | 125 | No |
+| anomaly\_capture\_bytesize | Anomaly Capture buffer byte size represents the maximum buffer size in terms of bytes during an anomaly capture | 0 B (disabled) | No |
+| anomaly\_capture\_duration | Anomaly Capture buffer duration represents the maximum span that the logs of an anomaly capture can belong to | 0s (disabled) | No |
+| capture\_flush\_mode | Sets the behavior of flushing captured contextual log buffers. Supported modes are listed below: <br> **local_per_source**: This is the default mode. captured buffer of a source is flushed when there's a local alert being triggered from same source. <br> **local_all**: This is the mode where all captured buffers are flushed when there's a local alert being triggered (not necessarily from same source). So in this mode whenever an alert is triggered from agent all capture buffers from all active sources will be flushed <br> **tag_per_source**: This is the mode where captured buffer of a source is flushed when there's an alert from same source and tag (from any agent within current tag) <br> **tag_all**: This is the mode where all captured buffers on all agents within the same tag is flushed whenever any of the agents trigger an alert| local_per_source | No |
 | grace\_period | Time the agent waits before triggering alerts. Can be in seconds \(s\) or minutes \(m\) | 0s | No |
-| _**log**_ **** | If omitted, agent uses default log name "edgedelta.log", default log level as "info" and default log path as installation path | N/A | No |
+| misc | Other miscellaneous mappings that you want to append to config | - | No |
+| ephemeral | It is only honored by clustering processor at the moment. 0.5 means 50% of a core. It can be enabled by setting cpu_friendly=true in clustering rule. | true | No |
```

Comment:
ephemeral and soft\_cpu\_limit seems swapped.

Reply:
Fixed it